style.css

```diff
@@ -0,0 +1,123 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	font-family: "Trebuchet MS", sans-serif;
+	font-size: 20px;
+	margin: 0;
+	padding: 0;
+	background-color: #e6f2f4;
+}
+
+nav {
+	font-size: 1.2rem;
+	height: 75px;
+	width: 100%;
+	background-color: #a7d9f4;
+	display: flex;
+	justify-content: space-around;
+	flex-direction: row;
+	align-items: center;
+}
+
+.navbar {
+	text-decoration: none;
+	color: black;
+}
+
+
+h1, h2, .me, .info, .projects {
+	text-align: center;
+	padding: 5px;
+}
```

Comment:
as mentioned in another comment, instead of individually styling these classes, it seems that you actually just want all p tags to have text-align: center

as a side note, .me and .info are both section tags. this means that you're setting text-align: center on a section tag, which is not directly targeting any text element. it is better in general to directly target the element you want to style, rather than its parent. this is because styling from the parent has quite low priority in css and can be easily overriden by other styling that more specifically targets the given element